### PR TITLE
Added bizkaibus.makdown

### PR DESCRIPTION
**Pull request by UgaitzEtxebarria**
**Description:**
Added the web page for the bizkaibus component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant/pull/22934

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/bizkaibus.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+date: 2019-04-22 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bizkaibus.png
+ha_category: Transport
+ha_iot_class: Cloud Polling
+ha_release: 0.93
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/bizkaibus.markdown]: Not necessary because this is not an existing page.

File: source/_components/bizkaibus.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+layout: page
+title: "Bizkaibus next bus tracking sensor"
+description: "Instructions on how to integrate timetable data for traveling on Bizkaibus within Home Assistant."
+date: 2019-04-22 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bizkaibus.png
+ha_category: Transport
```
Comment (klaasnicolaas) [inline source/_components/bizkaibus.markdown]:
```suggestion
ha_category: 
  - Transport
  - Sensor
```

File: source/_components/bizkaibus.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ha_release: 0.93
+redirect_from:
+ - /components/sensor.bizkaibus/
+---
+
+The `bizkaibus` sensor will give you the time until the next bus in the selected stop.
+
+The next website can help to determine the id of your bus stop. You can check if this is correct by going to 
+
+http://apli.bizkaia.net/APPS/DANOK/TQ/DATOS_PARADAS/DATOS_Paradas.xml (PR_CODRED is the STOP_ID)
```
Comment (klaasnicolaas) [inline source/_components/bizkaibus.markdown]: Maybe make this link clickable?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by UgaitzEtxebarria** [comment]
I forgot to add the logo inte source/images and I have included (I hope it is not too late)

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**